@navikt/core/css/link-card.css

```
@@ -0,0 +1,102 @@
+/* stylelint-disable declaration-block-no-redundant-longhand-properties */
+.navds-link-card {
+  --__ac-link-card-padding-block: var(--a-spacing-4);
+  --__ac-link-card-padding-inline: var(--a-spacing-5);
+
+  border-radius: var(--a-border-radius-xlarge);
+  text-decoration: none;
+  color: var(--a-text-default);
+  transition-property: border-color, box-shadow, transform, background-color;
+  transition-duration: 300ms;
+  transition-timing-function: cubic-bezier(0, 0, 0, 1);
+  position: relative;
+  background-color: var(--a-surface-default);
+  border: 1px solid var(--a-border-subtle);
+  padding: var(--__ac-link-card-padding-block) var(--__ac-link-card-padding-inline);
+  cursor: pointer;
+  display: grid;
+  grid-template-areas:
+    "image image"
+    "icon title"
+    "icon description"
+    "icon footer";
+  grid-template-columns: auto 1fr;
+  grid-template-rows: min-content min-content min-content auto;
+  align-items: center;
+}
+
+.navds-link-card:not(:has(.navds-link-card__description, .navds-link-card__footer)) {
+  grid-template-rows: auto 1fr;
+}
+
+.navds-link-card:not(:has(.navds-link-card__description, .navds-link-card__footer)) .navds-link-card__icon {
+  align-self: center;
+}
+
+.navds-link-card:hover {
+  border-color: var(--a-border-default);
+  box-shadow: 0 0 0 1px var(--a-border-default);
+}
+
+.navds-link-card__title {
+  grid-area: title;
+  color: var(--a-text-default);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: var(--a-spacing-2);
+  hyphens: auto;
+}
+
+.navds-link-card__description {
+  grid-area: description;
+  margin-block-start: var(--a-spacing-1);
+}
+
+.navds-link-card__footer {
+  grid-area: footer;
+  margin-block-start: var(--a-spacing-4);
+  display: flex;
+  align-self: flex-end;
+  gap: var(--a-spacing-2);
+  flex-wrap: wrap;
+}
+
+.navds-link-card__icon {
+  grid-area: icon;
+  width: max-content;
+  height: max-content;
+  display: grid;
+  place-content: center;
+  color: var(--a-text-default);
+  align-self: flex-start;
+  margin-inline-end: var(--a-spacing-4);
+}
+
+.aksel-link-card--small {
+  --__ac-link-card-padding-block: var(--ax-space-12);
+  --__ac-link-card-padding-inline: var(--ax-space-16);
```

Should we use old tokens here?
```suggestion
  --__ac-link-card-padding-block: var(--a-spacing-3);
  --__ac-link-card-padding-inline: var(--a-spacing-4);
```